### ISSUE TITLE
How to retrieve values with multiple barcode without tocuh event?

### ISSUE THREAD

**Issue by oumardiarra** [KingsMentor/MobileVisionBarcodeScanner#37]
Hi,
Thanks for the library, it works great. Why do we need to touch the screen in order to get result from multiple barcode scanning? is there a way to get them automatically like if we have only one barcode?

Best regards....

**Comment by KingsMentor** [comment]
@oumardiarra you can achieve this by

```java
barcodeCapture.setTouchAsCallback(false);
```

**Comment by oumardiarra** [comment]
Hi @KingsMentor 
thanks for the feedback. Where tu put that code. I've added it to the oncreate method like this:

` protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);
        BarcodeCapture barcodeCapture = (BarcodeCapture) getSupportFragmentManager().findFragmentById(R.id.barcode);
        barcodeCapture.setTouchAsCallback(false);
        barcodeCapture.setRetrieval(this);
        /* barcodeCapture.setTouchAsCallback(false);*/
    }`
and I'm still getting value when I only click on screen. Also I've added barcodeCapture.setShowFlash(true);
after barcode initialization but it doesn't work. Flash works only if i use it directly in fragment(app:gvb_flash="true)

**Comment by KingsMentor** [comment]
@oumardiarra don't forget to call :

```java
barcodeCapture.refresh(true);
```
call that method after changing any of the barcode preferences to make sure changes is reflected.
Let me know if this resolved the issue.

**Comment by oumardiarra** [comment]
@KingsMentor no,  I don't have the ability to add boolean parameter to  the refresh() method. 
Only `barcodeCapture.refresh();` and also using it , I have the same behavior. Do i have to call it before calling `barcodeCapture.setRetrieval(this);` or after? 
Though I have tested both with same result. Seems that only  Attributes on fragments works in my case but I don't have `setTouchAsCallback` on it.

**Comment by KingsMentor** [comment]
@oumardiarra update your dependency to : 

`compile 'xyz.belvi.mobilevision:barcodescanner:2.0.3'`

// forceUpdate can be true or false
Also, call `barcodeCapture.refresh(forceUpdate);` only after changing the barcode properties.

**Comment by KingsMentor** [comment]
@oumardiarra confirm if the issue has been resolved.

**Comment by oumardiarra** [comment]
@KingsMentor sorry for the delay. I always get values only when i click on screeen. I've updated my dependency as suggested above. Here is the code:

`barcodeCapture.setSupportMultipleScan(true);
        barcodeCapture.setShowDrawRect(true);
        barcodeCapture.setTouchAsCallback(false);
        barcodeCapture.refresh(true);`
